cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1,24 +1,269 @@
+                .collect();
+
+            Table::from_iter(packages)
+        } else {
+            // Add comment with example packages
+            let mut table = Table::new();
+
+            table.decor_mut().set_suffix(indoc! {r#"
+
+                # hello.pkg-path = "hello"
+                # nodejs = { version = "^18.4.2", pkg-path = "nodejs_18" }
+            "#});
+
+            table
+        };
+
+        install_table.decor_mut().set_prefix(indoc! {r#"
+            #
+            # This is a Flox environment manifest.
+            # Visit flox.dev/docs/concepts/manifest/
+            # or see flox-edit(1), manifest.toml(1) for more information.
+            #
+
+            # List packages you wish to install in your environment inside
+            # the `[install]` section.
+        "#});
+
+        manifest.insert(MANIFEST_INSTALL_TABLE_KEY, Item::Table(install_table));
+
+        // `[vars]` table
+        let mut vars_table = Table::new();
+
+        vars_table.decor_mut().set_prefix(indoc! {r#"
+            # Set environment variables in the `[vars]` section. These variables may not
+            # reference once another, and are added to the environment without first
+            # expanding them. They are available for use in the `[profile]` and `[hook]`
+            # scripts.
+        "#});
+        vars_table.decor_mut().set_suffix(indoc! {r#"
+
+            # message = "Howdy"
+        "#});
+
+        manifest.insert(MANIFEST_VARS_TABLE_KEY, Item::Table(vars_table));
+
+        // `[hook]` table
+        let mut hook_table = Table::new();
+
+        if let Some(ref hook_on_activate_script) = customization.hook_on_activate {
+            let on_activate_content = formatdoc! {r#"
+                """
+                {}
+                """"#
+            , indent::indent_all_by(2, hook_on_activate_script)};
+
+            hook_table.insert("on-activate", value(on_activate_content));
+        } else {
+            hook_table.decor_mut().set_suffix(indoc! {r#"
+
+                # on-activate = """
+                #     # Set variables, create files and directories
+                #     venv_dir="$(mktemp -d)"
+                #     export venv_dir
+                #
+                #     # Perform initialization steps, e.g. create a python venv
+                #     python -m venv "$venv_dir"
+                #
+                # """
+            "#});
+        };
+
+        hook_table.decor_mut().set_prefix(indoc! {r#"
+            # The `hook.on-activate` script is run by the *bash* shell immediately upon
+            # activating an environment, and will not be invoked if Flox detects that the
+            # environment has previously been activated. Variables set by the script will
+            # be inherited by `[profile]` scripts defined below. Note that any stdout
+            # generated by the script will be redirected to stderr.
+        "#});
```

Comment:
suggestion: move this up before populating the table so we dont set the prefix as the last step ^^